drivers/hmis/app/models/hmis/ce/referral_ce_event_manager.rb

```diff
@@ -62,13 +62,11 @@ def determine_event_type(capture_to_sentry: true)
 
       # Fall back to determining the event type based on the referral target project
       project = referral.target_project
-      event_type = HudUtility2026.project_to_ce_event_type(project)
-
+      event_type = HudHelper.util('2026').project_to_ce_event_type(project)
```

Comment:
I guess "2026" is called out since this helper exists in HudUtility2026 but not HudUtility2024. However the logic is not 2026-specific